### DISCUSSION TITLE
Overview of rights

### DISCUSSION THREAD

**Discussion by johanpeters** [akeeba/panopticon#430] [Ideas]
create a view based on every site, connected groups and members for every group so you get 1 overview of al the rights users and group have.
including the options defined for every group and every user.

**Comment by nikosdion** [comment]
That would be a very problematic view.

I know what you have in mind. Joomla's permissions debug page per user. That page work with Joomla! because you have a fairly small number of user groups and categories. Remember, Joomla! does not list permissions _per article_ (which is the equivalent of _per site_ in Panopticon).

Best tell me what you are trying to solve, not how you think you would go about solving it, and we'll figure out a better way to do it.
